packages/tx/package.json

```diff
@@ -20,7 +19,7 @@
     "tsc": "ethereumjs-config-tsc",
     "lint": "ethereumjs-config-lint",
     "lint:fix": "ethereumjs-config-lint-fix",
-    "test": "npm run test:node",
+    "test": "npm run test:node && npm run test:browser",
```

Comment:
OK. I might want to undo this in the future for some other optimizations, but that's super OK for now.